components/ca-object-list.html

```diff
@@ -9,11 +9,17 @@
     ca-object-list {
         display: block;
         position: relative;
-        overflow: hidden;
+        overflow: inherit;
```

Comment:
shouldn't need overflow: inherit - it should be implicit, just removing the hidden should work fine

Reply:
I'll try it

Reply:
Removed